Deleting non-existent logs causes an error

**MarkBind Version:** 1.7.2

**Description of Issue:**
Sometimes, when running markbind, the errors in the log below will appear.

This is because the program periodically clears the log files inside `_markbind/logs/`. However, these logs are still documented in `_markbind/logs/audit.json`, so the program will try to delete the log files again, but they no longer exist.

**Temporary Workaround:**
The current workaround is to manually delete `_markbind/logs/audit.json`.

**Log:**

```
warning: Error: EPERM: operation not permitted, stat 'C:\Users\Tan Wang Leng\Documents\GitHub\yamgent\markbind\test\test_site\_markbind\logs\markbind-2018-05-16.log'
error: EPERM: operation not permitted, stat 'C:\Users\Tan Wang Leng\Documents\GitHub\yamgent\markbind\test\test_site\_markbind\logs\markbind-2018-05-16.log'
fs.js:904
  return binding.readdir(pathModule._makeLong(path), options.encoding);
                 ^

Error: ENOENT: no such file or directory, scandir 'C:\Users\Tan Wang Leng\Documents\GitHub\yamgent\markbind\test\test_site\_site'
    at Object.fs.readdirSync (fs.js:904:18)
    at _walkSync (C:\Users\Tan Wang Leng\Documents\GitHub\yamgent\markbind\node_modules\walk-sync\index.js:74:18)
    at walkSync (C:\Users\Tan Wang Leng\Documents\GitHub\yamgent\markbind\node_modules\walk-sync\index.js:44:10)
    at Object.<anonymous> (C:\Users\Tan Wang Leng\Documents\GitHub\yamgent\markbind\test\test_site\testUtil\test.js:14:21)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
Test Failed
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! markbind-cli@1.7.2 testwin: `jest && cd test/test_site && test.bat`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the markbind-cli@1.7.2 testwin script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Tan Wang Leng\AppData\Roaming\npm-cache\_logs\2018-05-16T03_23_53_389Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -50,8 +50,8 @@
     "path-is-inside": "^1.0.2",
     "progress": "^2.0.0",
     "walk-sync": "^0.3.1",
-    "winston": "^2.4.1",
-    "winston-daily-rotate-file": "^3.0.1"
+    "winston": "^2.4.4",
+    "winston-daily-rotate-file": "^3.6.0"
   },
   "repository": {
     "type": "git",

